dist/index.html

```diff
@@ -24,23 +24,28 @@ <h1>OVERLOOK HOTEL</h1>
     </header>
   </nav>
   <main>
-    <section class="customer-details">
-      <!-- <h2 class="user-greeting">Hello Customer</h2> -->
-      <button class="future-trip-list trip-button">FUTURE TRIPS</button>
-      <button class="past-trip-list trip-button">PAST TRIPS</button>
-      <p class="total-expense">expenditure...$1,248</p>
-      <section class="room-display-wrapper">
-        <!-- BOXES GO HERE -->
+    <div class="main-page-layout">
+      <section class="log-in-wrapper">
```

Comment:
Are you wanting to make the login like a modal or overlay? Right now it looks like main-page-layout is not taking up the full width of the page, moving the log-in-wrapper into main or making main-page-layout wider might help!